pyrmq/publisher.py

```diff
@@ -104,6 +114,7 @@ def declare_queue(self, channel) -> None:
             exchange=self.exchange_name,
             routing_key=self.routing_key,
         )
+        channel.confirm_delivery()
```

Comment:
?

Reply:
According to [this](https://pika.readthedocs.io/en/stable/examples/blocking_publish_mandatory.html) it's a way to assure that messages are delivered. Personally I didn't see any difference in my tests but YMMV. We can remove this if you want.

Reply:
Okay, we can leave this.